go/mysql/collations/mysqldata.go

```diff
@@ -119,7 +119,7 @@ var tounicode_dec8_swedish_ci = [...]uint16{
 	0x00f8, 0x00f9, 0x00fa, 0x00fb, 0x00fc, 0x00ff, 0x0000, 0x0000,
```

Comment:
@shlomi-noach Any reason why the collations data was updated in this PR?

Reply:
@dbussink  yes! Per https://github.com/vitessio/vitess/pull/12551#pullrequestreview-1382840070, this is just a `gofmt` issue.

Reply:
I think the only change is to fix the lint warnings, removing the redundant `eightbit.UnicodeMapping{}` type.

Reply:
@dbussink  yes! Per https://github.com/vitessio/vitess/pull/12551#pullrequestreview-1382840070, this is just a `gofmt` issue.